relay-server/src/processing/events.rs

```diff
@@ -0,0 +1,416 @@
+//! Common functionality for transaction and event processing.
```

Comment:
What do you think of making this already a directory to follow the layout of the other modules?

Can also leave it just as it is and probably end up doing it with a later refactor.